src/components/Connect.js

```diff
@@ -16,7 +16,7 @@ export class Connect extends React.Component {
       return (
         <Button kind="googleDrive">
           <Spinner style={styles.spinners.toolbar}/>
-          Connecting to Google Drive...
+          Connecting to <br/>Google Drive...
```

Comment:
why is this necessary? Using `<br/>` tags is kind of an html code smell. Maybe you want this button not to be greater than a certain width? Then you should just set `max-width` on the style and the text will wrap for you.